modules/ROOT/pages/provisioning-hyperv.adoc

```diff
@@ -0,0 +1,104 @@
+= Provisioning Fedora CoreOS on Microsoft HyperV
+
+This guide shows how to provision new Fedora CoreOS (FCOS) nodes on Microsoft's HyperV.
+
+== Prerequisites
+
+You must have an Ignition configuration file containing your customizations. If you do not have one, see xref:producing-ign.adoc[Producing an Ignition File]
+You will also need a small utility from the libhvee https://github.com/containers/libhvee called `kvpctl`.  It "injects" your Ignition file into your virtual machine.  Precompiled binaries can be found on the project's https://github.com/containers/libhvee/releases[release page].
+
+=== Downloading the HDX image
+
+HDX images can be manually downloaded from the download page.  Be sure to decompress the image after downloading.
+
+== Booting a new VM on Microsoft HyperV
+
+=== Creating a virtual machine
+
+There are numerous ways to create a virtual machine in HyperV.  This approach uses the HyperV Manager and its new virtual machine wizard. Once you have the HyperV Manager graphic interface running, right-click on _New_ as seen in the following image.
+
+image::new_hyperv.png[]
+
+This will launch the _New Virtual Machine Wizard_.  You can follow the guided wizard but take note of the following two exceptions.
+
+==== Configuring networking
+
+When asked about the networking in the _New Virtual Machine Wizard_, be sure set up networking or your virtual machine will not be able to communicate with anything.  It is sufficient to select the _Default_ Networking option.
+
+==== Configuring the Fedora CoreOS image as the disk
+You need to assign the HDX disk image you downloaded earlier as the virtual machine's primary disk. This is not the default setting.  You must click _Use an existing virtual disk_ and then specify the location of the image you downloaded earlier.  See the image below for an example.
+
+image::hyperv-disk.png[]
+
+=== Setting the Ignition config
+
+Once your virtual machine is defined, do not start it.  Be sure to complete the following section before starting your virtual machine. When first booting Fedora CoreOS, you need to have an ignition file which is used to set up and configure the operating system.
+
+For the HyperV platform, Ignition will read its configuration file from the HyperV hypervisor in parts and then reassemble it.  Before this can occur, you can use `kvpctl` and its _add-ign_ subcommand to split your Ignition file and assign it to the virtual machine.  The syntax for the command is as follows:
+
+[source, bash]
+----
+> ./kvpctl.exe <name_of_vm> add-ign <path_to_ign_file> <ignition_config_prefix>
+----
+
+When Ignition is run on first boot, it will look for a pre-defined key name and reassemble the Ignition configuration file.  The pre-defined key name is *ignition.config.*.  Be sure to note the trailing `.` on the prefix key name.
+
+For example:
+
+[source, bash]
+----
+> .\kvpctl.exe podman-machine-default add-ign C:\Users\joe\myvm.ign ignition.config.
```

Comment:
```suggestion
[source, powershell]
----
.\kvpctl.exe podman-machine-default add-ign C:\Users\joe\myvm.ign ignition.config.
```

When using bash (Git Bash on Windows in this case) it's usually good form to provide Unix style paths. The example should still use Powershell I think.